ostd/src/io/io_mem/allocator.rs

```diff
@@ -25,17 +20,25 @@ pub(super) struct IoMemAllocator {
 }
 
 impl IoMemAllocator {
-    /// Acquires the I/O memory access for `range`.
+    /// Acquires `range` for insensitive MMIO.
     ///
     /// If the range is not available, then the return value will be `None`.
-    pub(super) fn acquire(&self, range: Range<usize>) -> Option<IoMem> {
-        debug!("Try to acquire MMIO range: {:#x?}", range);
+    pub(super) fn acquire(&self, range: Range<usize>) -> Option<IoMem<Insensitive>> {
+        debug!(
+            "Try to acquire range {:x?}..{:x?} for security-insensitive MMIO",
+            range.start, range.end
+        );
```

Comment:
The conflicts are resolved wrongly. Convert to the `("{:#x?}", range)` format instead.